profiles/models.py

```diff
@@ -1,3 +1,55 @@
 from django.db import models
+from django.core.validators import RegexValidator
 
-# Create your models here.
+from validation.validate_password import validate_password
+from validation.validate_phone_number import validate_phone_number_len, validate_phone_number_is_digit
+from validation.validate_edrpou import validate_edrpou
+from validation.validate_foundation_year import validate_foundation_year
+from validation.validate_image import validate_image_size, validate_image_format
+
+#REGIONS = ["east", "west", "north", "south"]
+REGIONS = (('E', "east"), ('W', "west"), ('N', "north"), ('S', "south"))
```

Comment:
isn't it more expedient to use a dictionary here?

Reply:
As we found, we have to use only tuple of tuples for field attribute 'choices'. It's on line 21 models.py. Are we wrong?

Reply:
indeed :(
as soon as we need choices here. let it be up to the next improvement.

Reply:
Thank you we'll try to find better solution for the next time.